src/components/AmplitudeAnalytics/index.ts

```diff
@@ -6,9 +6,8 @@ import { Identify, identify, init, track } from '@amplitude/analytics-browser'
  * Uniswap has two Amplitude projects: test and production. You must be a
  * member of the organization on Amplitude to view details.
```

Comment:
i also think this file should live in `src/analytics` rather than `src/components/AmplitudeAnalytics`